frontend/src/components/visual-editor/form/OptionsForm.tsx

```diff
@@ -104,6 +104,21 @@ export const OptionsForm = () => {
           name="options.fallback"
           control={control}
           defaultValue={block?.options?.fallback}
+          rules={{
+            validate: (value) => {
+              const hasNoValue = !value;
```

Comment:
I think block?.options?.fallback should always be an object { active: boolean; max_attempts: number, message: string[] }

Reply:
The rule should be : 

If max attempts > 0 => message should be supplied